lbry/blockchain/sync/claims.py

```diff
@@ -38,26 +38,46 @@ def channel_content_count_calc(signable):
 support = TXO.alias('support')
 
 
-def staked_support_aggregation(aggregate):
+def staked_support_subquery(claim_hash, aggregate):
```

Comment:
The first parameter should probably be `claim_hash_column` so as not to be confused with a `claim_hash` value.